src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    List<Order> getByUser(User user);
```

Comment:
it would be better to rename it 
```suggestion
    List<Order> getOrdersHistory(User user);
```